packages/lab/src/overlay/Overlay.tsx

```diff
@@ -21,15 +15,11 @@ import {
 
 export interface OverlayProps extends ComponentPropsWithoutRef<"div"> {
   open?: boolean;
-  onOpenChange?: (event: SyntheticEvent, newOpen: boolean) => void;
```

Comment:
Better to keep event whenever possible

Reply:
This was done to match other components in core

Reply:
(reopened this to check when reviewing) I see the event was removed, do we definitely want that re matching other components?

Reply:
That change does match the other components... `onOpenChange` everywhere else doesn't take an event.

Reply:
ahhh I was looking at an old commit haha ignore :)